src/coreclr/scripts/superpmi.py

```diff
@@ -588,6 +588,7 @@ def __collect_mc_files__(self):
             env_copy["SuperPMIShimLogPath"] = self.temp_location
             env_copy["SuperPMIShimPath"] = self.jit_path
             env_copy["COMPlus_AltJit"] = "*"
+            env_copy["COMPlus_AltJitNgen"] = "*"
```

Comment:
To be complete, you should add a line below that prints this (with the other print lines):
```
print_platform_specific_environment_vars(self.coreclr_args, "COMPlus_AltJitNgen", "*")
```